### PR TITLE
update logo

### PR DESCRIPTION
**Pull request by suzannejin**
- [x] move the nf-core logo a little to the right so that it can be seen better

## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/deepmodeloptim/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/deepmodeloptim _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core pipelines lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Check for unexpected warnings in debug mode (`nextflow run . -profile debug,test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: assets/nf-core-deepmodeloptim_logo_light.png
Comment (mashehu) [inline assets/nf-core-deepmodeloptim_logo_light.png:1]: something happened with the dash here... can you try to recreate it with `nf-core pipelines create-logo deepmodeloptim --format svg`?

Comment (suzannejin) [reply assets/nf-core-deepmodeloptim_logo_light.png:1]: that is exactly what i did.. create the logo using nf-core, and then add the cat-snack logo..
what is weird with the dash?

Comment (mashehu) [reply assets/nf-core-deepmodeloptim_logo_light.png:1]: It should have some space to "core" it looks like it is overlapping now.

Comment (suzannejin) [reply assets/nf-core-deepmodeloptim_logo_light.png:1]: weird... i tried to create-logo once again, and even the original logo has this weird dash problem...
I fixed it manually now though.. not sure if there is a better way?

Comment (mashehu) [reply assets/nf-core-deepmodeloptim_logo_light.png:1]: what vector graphics program are you using?

Comment (suzannejin) [reply assets/nf-core-deepmodeloptim_logo_light.png:1]: I use inkspace.. when i open the svg using inkspace it look all right after the edition, but on github everything still look over the place somehow

**Pull request closed without merging**